distributed/stealing.py

```diff
@@ -165,15 +165,11 @@ def move_task_request(self, key, victim, thief):
                          thief, self.scheduler.occupancy[thief])
 
             victim_duration = self.scheduler.processing[victim][key]
-            self.scheduler.occupancy[victim] -= victim_duration
-            self.scheduler.total_occupancy -= victim_duration
 
             thief_duration = self.scheduler.task_duration.get(key_split(key), 0.5)
             thief_duration += sum(self.scheduler.nbytes[key] for key in
                                   self.scheduler.dependencies[key] -
                                   self.scheduler.has_what[thief]) / BANDWIDTH
-            self.scheduler.occupancy[thief] += thief_duration
-            self.scheduler.total_occupancy += thief_duration
```

Comment:
Unfortunately it's important that we change the occupancy when we first try to steal.  This lets the load balancing machinery know when it should stop trying to move tasks around.

Reply:
Hmm. But this leads to undesirable behaviour elsewhere. 

What if `balance` were to look at an augmented version of occupancy, created by putting together info from occupancy and in_flight?

Reply:
> What if balance were to look at an augmented version of occupancy, created by putting together info from occupancy and in_flight?

That seems reasonable to me.  This would also allow us to uncomment the final check in `Scheduler.validate_state`.